datahub-web-react/src/app/search/SimpleSearchFilters.tsx

```diff
@@ -42,7 +45,9 @@ export const SimpleSearchFilters = ({ facets, selectedFilters, onFilterSelect, l
         onFilterSelect(newFilters);
     };
 
-    const sortedFacets = cachedProps.facets.sort((facetA, facetB) => {
+    const filteredFacets = cachedProps.facets.filter((facet) => FILTERS_TO_EXCLUDE.indexOf(facet.field) === -1);
```

Comment:
minor nit: could just use `.includes` here and that will be easier to read than `indexOf === -1`